### ISSUE TITLE
Missing fields in usa-cities.geojson 

### ISSUE THREAD

**Issue by MadGeometer** [ResidentMario/geoplot-data#1]
The usa-cities.geojson dataset is missing the names of the cities and towns. Can those be added, please?

**Comment by hartleybrody** [comment]
I agree! If I knew the original data source, I'd be happy to reprocess it and create a PR to include the city names. 

As it stands right now though, I can't use this data set for my project and had to grab a list that includes [far fewer US cities from this website](https://exploratory.io/map).

**Comment by chapmanjacobd** [comment]
[ne_us_cities.geojson.zip](https://github.com/ResidentMario/geoplot-data/files/7628522/ne_us_cities.geojson.zip)


```sh
# https://www.naturalearthdata.com/downloads/10m-cultural-vectors/10m-populated-places/

ogr2ogr ne_us_cities.geojson ne_10m_populated_places.dbf -dialect sqlite -where "ADM0_A3 = 'USA' "
~ $ wc -l ne_us_cities.geojson
776 ne_us_cities.geojson
```

Fewer cities than `usa-cities.geojson` but I previously spent about a year trying to curate my own list of global cities and eventually conceded to Natural Earth. The quality of the Natural Earth's dataset is very consistent and evenly distributed where you don't have 40+ "cities" in St. Louis and one datapoint for another city of similar population.
